src/lib/mdns/Resolver.h

```diff
@@ -118,10 +118,10 @@ struct DiscoveryFilter
 {
     DiscoveryFilterType type;
     uint16_t code;
-    char * instanceName;
+    const char * instanceName;
```

Comment:
Why do we need discovery by instance name? Can't a regular "resolve" be used when the instance name is known?

Reply:
took this discussion to slack with Damian, will create a separate issue if we decide there is another approach that makes sense. In this PR, the change made is code cleanup and not architectural.